Client/Frontend/Browser/PageZoom/PageZoomView.swift

```diff
@@ -0,0 +1,208 @@
+// Copyright 2022 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import SwiftUI
+import BraveUI
+import WebKit
+import Data
+import Shared
+import BraveShared
+
+private struct ZoomView: View {
+  private let color = Color(UIColor.braveLabel)
+  
+  var minValue: Double
+  var maxValue: Double
+  @Binding var value: Double
+  
+  var onDecrement: () -> Void
+  var onReset: () -> Void
+  var onIncrement: () -> Void
+  
+  var body: some View {
+    HStack(spacing: 0.0) {
+      Button(action: {
+        onDecrement()
+      }) {
+        Image(systemName: "minus")
+          .font(.system(.footnote).weight(.medium))
+          .foregroundColor(color)
+          .frame(width: 44.0)
+          .padding(.vertical, 8.0)
+          .contentShape(Rectangle())
+      }
+      .fixedSize()
+      .disabled(value == minValue)
+      
+      Divider()
+        .padding(.vertical, 8.0)
+      
+      Text(PageZoomView.percentFormatter.string(from: .init(value: 0.999)) ?? "999%")
+        .font(.system(.footnote).weight(.medium))
+        .fixedSize()
+        .hidden()
+        .overlay(resetZoomButton)
+        .padding(.horizontal)
+        .padding(.vertical, 8.0)
+      
+      Divider()
+        .padding(.vertical, 8.0)
```

Comment:
You have 8pt's of vertical padding on every child, therefore you should remove it from all children and add it to the parent instead